src/Main.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './App.css';
+import HornedBeast from './HornedBeast';
+
+let testArray = [{
```

Comment:
So it looks like you added the array but didn't do anything with it? Did you have an ideas about how to make this stretch goal work?